src/test/java/AddCommandTest.java

```diff
@@ -0,0 +1,86 @@
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import java.util.InputMismatchException;
+
+class AddCommandTest {
+
+    @Test
+    void shouldPrintInfoOfUserThatHasBeenAddedToUserbook() {
+        User user = new User("test", "test", 200);
+
+        FakeUserbook usersBook = new FakeUserbook(user);
+        FakeConsole console = new FakeConsole("test", 200);
+
+        Command command = new AddCommand(usersBook, console);
+
+        boolean isConsoleEmpty = console.messages.isEmpty();
+        Assertions.assertTrue(isConsoleEmpty);
+
+        command.execute();
+        Assertions.assertTrue(usersBook.userAdded, "method Add is not called");
+
+        boolean isInfoMessageAboutAddingUserIsPresent = console.messages
+                .stream()
+                .anyMatch(message -> message.startsWith
+                        ("New user created: Name = 'test', Surname = 'test', Age = 200'"));
+
+        Assertions.assertTrue(isInfoMessageAboutAddingUserIsPresent, "Console-message is Empty");
+
+
```

Comment:
you should also check that proper user was added to userBook. Here you test that user was printed to console. But you have 2 actions:  save user to book and print him